### ISSUE TITLE
Electron does not start with electron-is-dev 3.0.0+

### ISSUE THREAD

**Issue by malatin3** [adopted-ember-addons/ember-electron#1686]
Adding ember-electron to a fresh ember app and attempting to run it results in this error:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module ..../electron-app/node_modules/electron-is-dev/index.js from ..../electron-app/src/index.js not supported.
Instead change the require of ..../electron-app/node_modules/electron-is-dev/index.js in ..../electron-app/src/index.js to a dynamic import() which is available in all CommonJS modules.
```
Was able to fix it by downgrading electron-is-dev from 3.0.1 to 2.0.0

**Comment by bendemboski** [comment]
This was addressed in #1640, which hasn't been released, but that wouldn't really help existing apps anyway since it's a change to the blueprint.

The upshot is that `electron-is-dev` is no longer necessary, so you should remove it from your dependencies and use [app.isPackaged](https://www.electronjs.org/docs/latest/api/app#appispackaged-readonly) instead.

**Comment by malatin3** [comment]
Makes sense! ty
